test/testmie.f90

```diff
@@ -0,0 +1,169 @@
+!   This file is part of EmDee.
```

Comment:
This file is not necessary. Moreover, its name does not fit in the convention for EmDee's testing platform (the name of testing codes must start with test_).